app/models/assignment.rb

```diff
@@ -62,6 +65,16 @@ def self.needs_check_in
         CHECK_IN_INTERVAL.ago, CHECK_IN_INTERVAL.ago)
   end
 
+  def missed_check_ins?
+    return false unless status == STATUSES[:accepted] &&
+      project.status == Project::STATUSES[:in_process_underway] &&
+      last_contacted_at < UNRESPONSIVE_INTERVAL.ago
+
+    check_ins = notes.order(created_at: :desc).limit(MISSED_CHECK_INS)
+    return false unless check_ins.count == MISSED_CHECK_INS
```

Comment:
Super minor but I would use `>=` here instead of `==` just in case we have an off-by-one or bookkeeping error somewhere in the future